packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -428,6 +428,27 @@ export class RestApi extends Resource implements IRestApi {
   }
 }
 
+/**
+ * The EndpointConfiguration property type specifies the endpoint types of a REST API.
+ *
+ * EndpointConfiguration is a property of the AWS::ApiGateway::RestApi resource.
+ */
+export interface EndpointConfiguration {
+  /**
+   * A list of endpoint types of an API or its custom domain name.
+   *
+   * @default - no endpoint types.
+   */
+  readonly types: EndpointType[];
+
+  /**
+   * A list of VPC endpoint IDs of an API against which to create Route53 ALIASes
+   *
+   * @default - no ALIASes are created for the endpoint.
+   */
+  readonly vpcEndpointIds: string[];
```

Comment:
I believe the type for this should be the ec2 module's [IVpcEndpoint](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ec2.IVpcEndpoint.html).

Reply:
Thanks for the feedback!